recipes/default.rb

```diff
@@ -51,15 +51,26 @@
   action :nothing
 end
 
-template '/etc/init/chef-guard.conf' do
-  source 'chef-guard.upstart.erb'
-  backup false
-  variables(
-    :recipe_file => (__FILE__).to_s.split("cookbooks/")[1],
-    :template_file => source.to_s,
-    :basedir => node['chef-guard']['install_dir']
-  )
-  notifies :restart, 'service[chef-guard]'
+# Check if CentOS/RHEL 7 is used, and thus assume systemd is present
+if node['platform_family'] == 'rhel' && node['platform_version'].to_i >= 7 && node['platform_version'].to_i < 8
```

Comment:
Well, historically, RHEL 5 was sysv, RHEL 6 upstart, RHEL 7 systemd, RHEL8 ... Just a precaution I guess, but your right that at there is no real reason for that atm ;)